ArduCopter/GCS_Mavlink.cpp

```diff
@@ -880,6 +880,14 @@ void GCS_MAVLINK_Copter::handle_mount_message(const mavlink_message_t &msg)
 
 void GCS_MAVLINK_Copter::handleMessage(const mavlink_message_t &msg)
 {
+    // for mavlink SET_POSITION_TARGET messages
+    constexpr uint32_t MAVLINK_SET_POS_TYPE_MASK_POS_IGNORE      = ((1<<0) | (1<<1) | (1<<2));
```

Comment:
maybe we can use https://mavlink.io/en/messages/common.html#POSITION_TARGET_TYPEMASK

Reply:
Done